fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -250,12 +286,12 @@ const CreateElection = () => {
       {questions.map((value, idx) => (
         // FIXME: Do not use index in key
         // eslint-disable-next-line react/no-array-index-key
-        <View key={idx.toString()}>
+        <View key={idx.toString()} style={{ marginBottom: Spacing.x2 }}>
```

Comment:
Inline styles are discouraged, I would create a small StyleSheet at the top of the file with a `questionWrapper` style :)